### PR TITLE
Added debts API

### LINKED ISSUE #214: Expose debts through API
Implement an API endpoint that would display list of debts. This API will be consumed in dashboard. Should work on both exit nodes, and client nodes.

# Example

Request

```
curl -H'Accept: application/json' http://127.0.0.1:12345/api/debts
```

Example response

```json
{
  "fd80::f00": 12345,
  "fd80::f01": 12346
}
```

Keys are the mesh IPs, and values are debt amounts.

It could be a up to a consumer how to display this data. For example it could be sorted by debt value, and IP address.

### PR DESCRIPTION
**Pull request by mpapierski**
This PR implements debts API. Unfortunately we can't just simply transform `HashMap<K, V>` into a JSON object when `K` is a struct. Because of that this API returns a list of tuples similiar to `Vec<(K, V)>`.

Closes #214

### CODE REVIEW

File: docs/api/router-dashboard.md [hunk 1]
```diff
@@ -531,3 +531,36 @@ Calling HTTP `DELETE` request on this endpoint causes all tables to be wiped out
...
+  - Code: 200 OK
+  - Contents: `JSON` structured message. See below for an example format.
+- Error Response: `500 Server Error`
+- Sample Call
+
+`curl 127.0..1:<rita_dashboard_port>/debts`
+
+Format:
+```
+[
```
Comment (jkilpatr) [inline docs/api/router-dashboard.md]: is this in json style with field names or just the data?

File: rita/src/client.rs [hunk 2]
```diff
@@ -229,6 +229,7 @@ fn main() {
             ).route("/info", Method::GET, get_own_info)
             .route("/version", Method::GET, version)
             .route("/wipe", Method::POST, wipe)
+            .route("/debts", Method::GET, get_debts)
```
Comment (jkilpatr) [inline rita/src/client.rs]: why not in the rita_common endpoints?

File: rita/src/exit.rs [hunk 3]
```diff
@@ -236,6 +236,7 @@ fn main() {
             .route("/version", Method::GET, version)
             .route("/wipe", Method::POST, wipe)
             .route("/database", Method::DELETE, nuke_db)
+            .route("/debts", Method::GET, get_debts)
```
Comment (jkilpatr) [inline rita/src/exit.rs]: same as for clients

File: docs/api/router-dashboard.md (hunk 1 above)
Comment (mpapierski) [reply docs/api/router-dashboard.md]: Yeah. Actually both keys, and values are dictionaries. Updating right now.

File: rita/src/exit.rs (hunk 3 above)
Comment (jkilpatr) [reply rita/src/exit.rs]: this is me being confused, ignore me.

File: docs/api/router-dashboard.md (hunk 1 above)
Comment (jtremback) [reply docs/api/router-dashboard.md]: @mpapierski could we have it be an object?

```json
{
   identity: {...}
   payment_details: {...}
}
```

Comment (mpapierski) [reply docs/api/router-dashboard.md]: Good point. Not sure why I didn't think of it before. Feels much better. Thanks!

File: docs/api/router-dashboard.md [hunk 4]
```diff
@@ -549,17 +549,22 @@ Calling HTTP `GET` request on this endpoint returns a list of debts.
...
-   debt_buffer]],
-  ...
+  [
+    {
+      "mesh_ip": "a:b:c:d:e:f:g:h",
+      "eth_address": "0x0101010101010101010101010101010101010101",
+      "wg_public_key": "pubkey"
+    },
+    {
+      "total_payment_recieved": "0x0",
```
Comment (asoltys) [inline docs/api/router-dashboard.md]: should be received not recieved

Comment (mpapierski) [reply docs/api/router-dashboard.md]: Thanks for pointing out @asoltys. I'll run `sed` to replace all occurrences globally in repo.

**Pull request merged**